app/assets/javascripts/sign_up_async_request.js.erb

```diff
@@ -95,32 +86,52 @@ function validateEmailDomain(email){
 	var domain = email.split('@')
 	if(domain[1] != "schools.nyc.gov"){
 		$(".spinner").css("display","none");
-	    $('#check_email_div').css("display","none");
+		$('#check_email_div').css("display","none");
 		throw new InvalidEmailDomainError("Invalid e-mail domain")
 	}
 }
 
 
 window.ClientSideValidations.callbacks.element.fail = function(element, message, callback) {
 	callback();
+	$("#sign_up_button").css("background-color","grey");
```

Comment:
A linter would recommend adding a space after the comma.

Even though it's gray, is the button still enabled?

Reply:
Removed that, we didn't need it.